vpr/src/analytical_place/detailed_placer.cpp

```diff
@@ -75,6 +75,15 @@ AnnealerDetailedPlacer::AnnealerDetailedPlacer(const BlkLocRegistry& curr_cluste
         }
     }
 
+    // The solution produced by the AP flow is significantly better than the
+    // initial placement solution produced by the initial placer in the default
+    // flow. Even though the annealer auto-selects its initial temperature based
+    // on the quality of the placement, we found that the initial temperatute was
+    // still too high and it was hurting quality. This scales down the initial
+    // temperature auto-selected by the annealer to prevent this and improve
+    // runtime.
+    float anneal_auto_init_t_scale = vpr_setup.PlacerOpts.place_auto_init_t_scale * 0.5f;
```

Comment:
Explain that we use an init_t that is half as big, and still scale by what the user sent in.